ViAn/Project/project.cpp

```diff
@@ -110,7 +110,7 @@ void Project::set_name_and_path(const std::string& name, const std::string& path
     m_name = name;
     if (!path.empty()){
         // Update path for all VideoProjects
-        m_dir = path + "/" + name + "/";
+        m_dir = path + name + "/";
```

Comment:
Will path always end with a `/`? Otherwise it might be worth checking that it does

Reply:
I needed to remove a '/' to get the open folder function to work. My guess is that the path saved in the code and memory weren't identical. Currently an extra '/' is added in projectdialog ok_btn_clicked so it will always have at least one. And the functions work whenit has multiple